src/typeDefs/dbSchema.ts

```diff
@@ -26,6 +26,8 @@ export type Submission = Omit<gqlType.Submission, 'facility' | 'healthcareProfes
     healthcareProfessionals: gqlType.HealthcareProfessionalSubmission[]
 }
 
+export type AuditLogs = Omit<gqlType.AuditLogs, 'AuditLogs'>
+
```

Comment:
@ermish Can you briefly explain what this file exactly does? Is it the types that are stores in the DB? Not sure if I fully understand this part.

Reply:
I added some comments around how the file works, but I guess I didn't explain what it was 😓 
these types are meant to match the data that is saved in the database.

It's not guaranteed that the data object structure is identical in front-end, api calls, backend logic/services, and in the database. for example, you don't want security data in your api objects, but you do in your service logic. Similarly, the way you save your data might be different (normalized vs denormalized for example) than how you use the data in your service. 

The Omit<> type is a shortcut instead of making a whole new type object, you can  just remove certain properties from an existing one.